apps/nova/src/components/Common/ForgetPassword.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Controller,
+  Dialog,
+  Field,
+  Label,
+  toaster,
+  Form,
+  useFormatMessage,
+  Button,
+  ButtonType,
+  ButtonSize,
+} from "@litespace/luna";
+import { IUser, Void } from "@litespace/types";
+import { useCallback, useMemo } from "react";
+import { useForm } from "react-hook-form";
+import { useForgetPassword } from "@litespace/headless/auth";
+
+type ForgetPasswordProps = {
+  open: boolean;
+  close: Void;
+};
+
+interface IForm {
+  email: string;
+  callbackUrl: string;
+}
+
+const ForgetPassword = ({ open, close }: ForgetPasswordProps) => {
+  const intl = useFormatMessage();
+
+  const { control, watch, formState, handleSubmit, reset } = useForm<IForm>({
+    mode: "onSubmit",
+    defaultValues: {
+      callbackUrl: "http:/localhost:5173/reset-password",
```

Comment:
`callbackUrl` is not part of the form. The client should construct it from the current site URL. 

```ts
const origin = location.origin //  this can be "http://localhost:5173" (local) or "https://..vercel.app" (staging) or "https://app.litespace.org" (production)
const callbackUrl = origin.concat(Route.ResetPassword) // send this to the server
```

The `callbackUrl` is used to redirect the user to the right URL based on the environment.